package.json

```diff
@@ -18,27 +18,34 @@
   "scripts": {
     "blc:suspensive.org": "pnpm blc https://suspensive.org --ro",
     "build": "turbo run build",
-    "build:watch": "turbo run build:watch --parallel",
+    "build:watch": "turbo watch build",
```

Comment:
I'm curious if there is a specific reason why the build part is not named ci:build.